tests/page-objects/dashboard/DashboardPage.ts

```diff
@@ -128,6 +134,9 @@ class DashboardPage extends BasePage {
         chartGridItemContainer: `${slct(COMMON_DASH_SELECTORS.DASH_GRID_ITEM)} .chartkit`,
         dashPluginWidgetBody: slct('chart-widget'),
         dashkitGridItem: slct('dashkit-grid-item'),
+
+        chartResetButton: '.widget-header__filters-controls .yc-button',
```

Comment:
Let's create a new qa for this button and pass it to `Button` in `WidgetHeader`, please